scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -121,6 +121,13 @@
 <div class="container-fluid">
   {{ pagination_footer(more_variants, page) }}
 </div>
+<!--  Pedigree_panel -->
+{% set has_pedigree = case.madeline_info and case.individuals|length > 1 %}
+{% if has_pedigree %}
+<div class="row">
+    <div class="col-md-4">{{ pedigree_panel(case) }}</div>
+  {% endif %}
```

Comment:
hihi almost - the end div needs to go inside as well.. 😉

Reply:
😳